src/app/ui/color_wheel.cpp

```diff
@@ -336,16 +352,32 @@ void ColorWheel::onOptions()
   menu.addChild(&tetradic);
   menu.addChild(&square);
 
-  if (isDiscrete()) discrete.setSelected(true);
-  switch (m_harmony) {
-    case Harmony::NONE: none.setSelected(true); break;
-    case Harmony::COMPLEMENTARY: complementary.setSelected(true); break;
-    case Harmony::MONOCHROMATIC: monochromatic.setSelected(true); break;
-    case Harmony::ANALOGOUS: analogous.setSelected(true); break;
-    case Harmony::SPLIT: split.setSelected(true); break;
-    case Harmony::TRIADIC: triadic.setSelected(true); break;
-    case Harmony::TETRADIC: tetradic.setSelected(true); break;
-    case Harmony::SQUARE: square.setSelected(true); break;
+  if (m_colorModel == ColorModel::NORMAL_MAP) {
```

Comment:
If the color model is `NORMAL_MAP`, we could just avoid adding `none`, `complementary`, `monochromatic`, etc. menu items inside `menu` variable.